e2e/specs/add_rows.spec.ts

```diff
@@ -35,8 +35,7 @@ describe("st.add_rows", () => {
   it("works for all elements that support it", () => {
     cy.get(".element-container .stTable").should("have.length", 3);
     cy.get(".element-container .stDataFrame").should("have.length", 4);
-    cy.get(".element-container .stChart").should("have.length", 3);
-    cy.get(".element-container .stVegaLiteChart").should("have.length", 12);
+    cy.get(".element-container .stVegaLiteChart").should("have.length", 15);
   });
```

Comment:
Add a test that you can start with an empty `st.line_chart` and then `add_rows` to it